Sources/WhoopDIKit/Container/Container.swift

```diff
@@ -121,9 +126,40 @@ public final class Container {
     }
 
     private func getDefinition(_ serviceKey: ServiceKey) -> DependencyDefinition? {
-        localDependencyGraph.acquireDependencyGraph { localServiceDict in
-            return localServiceDict[serviceKey] ?? serviceDict[serviceKey]
-        }
+        serviceDict[serviceKey]
```

Comment:
do we not need to grab the graph here?

Reply:
Nope - we no longer have a mutable `localServiceDict` that we need to protect. The only mutations which should be happening in the container now are within local scopes.